rules/windows/process_creation/proc_creation_win_susp_microsoft_onenote_child_process.yml

```diff
@@ -1,12 +1,17 @@
+---
```

Comment:
```suggestion

```
https://yaml.org/spec/1.2.2/ "YAML uses three dashes (“---”) to separate [directives](https://yaml.org/spec/1.2.2/#directives) from [document](https://yaml.org/spec/1.2.2/#documents) [content](https://yaml.org/spec/1.2.2/#nodes)."
It is not usefull in a sigma rule